apps/clinic/api/views.py

```diff
@@ -62,10 +66,17 @@ def resend(self, request, uuid):
         try:
             auth.send_otp_code()
         except SendSMSException:
-            return Response(data={"error": "خطایی در ارسال پیامک. لطفا با پشتیبان تماس بگیرید"}, status=status.HTTP_400_BAD_REQUEST)
+            return Response(data={"error": "خطایی در ارسال پیامک. لطفا با پشتیبان تماس بگیرید"},
+                            status=status.HTTP_400_BAD_REQUEST)
         return Response(status=status.HTTP_200_OK)
 
     def _get_auth(self, uuid):
-        return get_object_or_404(AuthSMSRequest, uuid=uuid) 
+        return get_object_or_404(AuthSMSRequest, uuid=uuid)
+
+
+class WorkSampleApiView(PermissionRequireMixin, APIView):
+    permissions = [User.Roles.SECRETARY, User.Roles.DOCTOR, User.Roles.ADMIN, User.Roles.PATIENT]
 
-        
+    def get(self, request):
+        work_sample = WorkSample.objects.filter(is_published=True).values_list()
+        return Response(data={'model_to_reeturn':list(work_sample)}, status=status.HTTP_200_OK)
```

Comment:
model_to_return isn't a meaningful name, please use a good name for that. Something like: `works` or `work_samples`